addons/mail/views/mail_canned_response_views.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+
+        <record id="mail_canned_response_view_search" model="ir.ui.view">
+            <field name="name">mail.canned.response.search</field>
```

Comment:
:socks: See guidelines about naming (to propagate to other views)
```suggestion
            <field name="name">mail.canned.response.view.search</field>
```

Reply:
Could you please provide those guidelines?

Reply:
Sur https://www.odoo.com/documentation/17.0/contributing/development/coding_guidelines.html#security-view-and-action